modules/ROOT/pages/batch-filters-and-batch-aggregator.adoc

```diff
@@ -159,15 +146,20 @@ You can, for example, for example, use the Groovy scripting module to modify the
 </batch:job>
 ----
 
-The sequential access method assumes that:
+//TODO: NEED TO EXPLAIN THE FOLLOWING BULLET POINTS.
```

Comment:
TODO: Clarify the bullet points below:

Sequential access to records assumes the following:
. The aggregator size matches the number of aggregated records.
. There is a direct correlation between the aggregated records and the items in the list.